scripts/convert_vep_vcf_to_tsv_capice.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+# Stops script if any error occurs.
+set -e
+
+# Defines error echo.
+errcho() { echo "$@" 1>&2; }
+
+# Usage.
+readonly USAGE="VEP VCF output to CAPICE TSV converter
+Usage:
+convert_vep_to_tsv_capice.sh -i <arg> -o <arg>
+-i    required: The VEP output VCF
+-o    required: The directory and output filename for the CAPICE .tsv.gz
+
+Example:
+bash convert_vep_vcf_to_tsv_capice.sh -i vep_out.vcf -o capice_in.tsv.gz
+
+Requirements:
+BCFTools
+"
+
+main() {
+  digestCommandLine "$@"
+  processFile
+}
+
+digestCommandLine() {
+  while getopts i:o:h flag
+  do
+    case "${flag}" in
+      i) input=${OPTARG};;
+      o) output=${OPTARG};;
+      h)
+        echo "${USAGE}"
+        exit;;
+      \?)
+        errcho "Error: invalid option"
+        echo "${USAGE}"
+        exit 1;;
+    esac
+  done
+
+  validateCommandLine
+}
+
+validateCommandLine() {
+  local valid_command_line=true
+
+  # Validate if variable is set & not empty.
+  if [ -z "${input}" ]
+  then
+    valid_command_line=false
+    errcho "input file not set/empty"
+  else
+    # Validate if input file exists.
+    if [ ! -f "${input}" ]
+    then
+      valid_command_line=false
+      errcho "input file does not exist"
+    else
+      # Validate allowed input filetype.
+      case $(file --mime-type -b "${input}") in
+        text/plain);;
+        application/*gzip);;
+        *)
+          valid_command_line=false
+          errcho "input file has invalid type (plain text/gzip allowed)";;
+      esac
+    fi
+  fi
+
+  # Validate if variable is set & not empty.
+  if [ -z "${output}" ]
+  then
+    valid_command_line=false
+    errcho "output file not set/empty"
+  else
+    # Validates proper output filename.
+    if [[ "${output}" != *.tsv.gz ]]
+    then
+      valid_command_line=false
+      errcho "output filename must end with '.tsv.gz'"
+    else
+      # Validates if output doesn't file already exist.
+      if [ -f "${output}" ]
+      then
+        valid_command_line=false
+        errcho "output file already exists"
```

Comment:
I would expect the following behavior: Running with -o output.tsv fails if the output file already exists 'output.tsv already exists, use -f to overwrite'. It should not create an output file with a different name.